rust/lesson-plan.md

```diff
@@ -0,0 +1,315 @@
+
+- The #beginners channel on the official [Rust Discord]. You are almost
+  guaranteed to get some answer here, and if not, don't hesitate to ask again.
+  The people who hang out here are there specifically to help. Because of time
+  zone differences it may take time for somebody to respond. Only English will
+  be consistently understood here.
+
+- The QQ Rust group ([QR code][qq]). For Chinese students, this is one of the
+  major Rust communities in China. This is a great place to hang out generally,
+  and for those unconfident in their English skills, a great place to ask
+  for help.
+
+- The WeChat Rust group #2 ([QR code][wechat]). Again this is mostly for the
+  Chinese audience. With WeChat's group limits there are multiple Rust groups.
+  This is the most recent. It is though relatively unpopulated, so you are more
+  likely to get help in the QQ group. Do join this one though to get in before
+  it spills over into a third group.
+
+These resources may also be helpful:
+
+- The official [users forum]. Apply the "help" tag to your post. Questions
+  usually recieve an answer here, but the responses can be limited.
+
+- [StackOverflow]. Apply the "rust" tag. You may or may not recieve a satisfying
+  answer.
+
+You are also welcome to email the primary author of this course, [Brian
+Anderson][brson], at brian@pingcap.com. I will happily answer questions, and am
+eager to hear your experience with the course.
+
+Finally, if there is a [Rust user group] near you, go check it out. As a Rust programmer
+these groups are where you will build some of your strongest connections. (Note
+that that link goes to the old Rust website and may not be up to date).
+
+
+## Making PNA Rust better
+
+At the end of each project is a link to a survey about your experience. Please
+take a few minutes to complete it, and be comfortable expressing the challenges
+you faced, and your criticisms of the course. The survey results are not seen by
+anybody but the [primary author of the course][author], though aggregate
+statistics may be reported publicly.
+
+As you work through the course content, please be on the lookout for things you
+would improve about the course, and either [submit issues][si] explaining, or
+[submit pull requests][spr] with improvements. (If you are being graded,
+accepted pull requests count toward extra credit during final evaluation &mdash;
+let your instructor or evaluator know! Accepted pull requests to any other
+project used during this course count as well. Note: I have no way to guarantee
+this is true…). This is an opportunity to gain experience contributing to an
+open-source Rust project. Make this a better course for the next person to take
+it than it was for you.
+
+See [CONTRIBUTING.md] for more.
+
+
+## Practical Networked Applications in Rust
+
+This is an outline of the course. Clicking each header will take you to the
+instructions for that section.
+
+### [Building Blocks 1][b1]
+
+Rust tooling.
```

Comment:
This feels short, maybe we can put a motivating paragraph here in the future, it's not a blocker though. :)

Reply:
Yeah it is. And I expect as I actually pull in background readings this will change.